src/utils/size.scss

```diff
@@ -1,4 +1,4 @@
-@import "../base/config.scss";
+@import "../config/config.scss";
```

Comment:
It seems like anything previously importing `base/index.scss` should import `config/index.scss`, as that's going to pull in the functions and mixins, as well as the new `base/index.scss` to get the reset and typography styles. Does that seem right?